Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerRmiController.java

```diff
@@ -89,6 +90,60 @@ public void changeHost(String roomName, String host) {
         }
     }
 
+    public void addGoals(String username, int goals) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)